.gitignore

```diff
@@ -114,3 +114,19 @@ pycodestyle*.py
 
 # For IDE settings
 .vscode
+
+# For Terraform
+**/.terraform/*
+*.tfstate
+*.tfstate.*
+*.terraform.lock*
```

Comment:
it's better to not ignore `.terraform.lock.hcl`: https://www.terraform.io/language/files/dependency-lock#lock-file-location

Reply:
Thanks, I removed this line and I added `.terraform.lock.hcl` files to the repository.